common/src/test/java/io/seata/common/loader/EnhancedServiceLoaderTest.java

```diff
@@ -54,15 +54,15 @@ public void testLoadException() {
     @Test
     public void testLoadByClass() {
         Hello load = EnhancedServiceLoader.load(Hello.class);
-        assertThat(load.say()).isEqualTo("Bonjour");
+        assertThat(load.say()).isEqualTo("Olá.");
     }
 
     /**
      * Test load by class and activate name.
      */
     @Test
     public void testLoadByClassAndActivateName() {
-        Hello englishHello = EnhancedServiceLoader.load(Hello.class, "EnglishHello");
+        Hello englishHello = EnhancedServiceLoader.load(Hello.class,"EnglishHello");
```

Comment:
Should keep format unchanged as before, missing space character here.

Reply:
Restored the missing space character.